pkg/security/rules/policy_monitor.go

```diff
@@ -75,6 +75,19 @@ func (p *PolicyMonitor) SetPolicies(policies []*rules.Policy, mErrs *multierror.
 	}
 }
 
+// ReportHeartbeatEvent sends HeartbeatEvents reporting the current set of policies
+func (p *PolicyMonitor) ReportHeartbeatEvent(sender events.EventSender) {
+	log.Infof("send heartbeat")
```

Comment:
Should this be a log.Info? or a log.Debug?

Reply:
I kept the log level set by @Malo10LeGoff in his initial PR but I agree that it would probably be less noisy to change this to Debug.
Unfortunately I already merged this PR so feel to open another to change this.